Add support for remote port, continous test, tiny interval and os signal

orn1983 opened the pull request:
Add a few useful features:
* Support for sub-second intervals by using floating points.
* Configurable port via flag -remote-port for twamp setup
* Graceful handling of os interrupt signals which will still return statistics
* Run test in continuous mode, stoppable via os interrupts

orn1983 commented:
Ah, slight change required if you merge the PR. I had to change the path of the import of the twamp-client while developing. You would have to change the path back if you merge.

tcaine commented:
> Ah, slight change required if you merge the PR. I had to change the path of the import of the twamp-client while developing. You would have to change the path back if you merge.

Can you update this in the PR?  I'd like to pull in these changes.

orn1983 commented:
All right! Fixed a few lingering bugs for edge cases, updated the README and fixed the import path back to your repo.

Thanks for merging. Sorry about some unnecessary changes (function names and such). I thought you weren't going to approve the PR so I just went a bit rogue.
